File: client/Client/Network/Messages/Session/AcceptCreateSessionMessage.cs
using System;

namespace LoadRunnerClient.Messages
{
    public class AcceptCreateSessionMessage
    {
        public const string TYPE = "AcceptCreateSessionMessage";

        public AcceptCreateSessionMessage()
        {
        }

        public AcceptCreateSessionMessage(String sessionID)
        {
            this._sessionID = sessionID;
        }

        private string _sessionID;

        public string sessionID
        {
            get => _sessionID;
            set => _sessionID = value;
        }

        private string _color;

        public string color
        {
            get => _color;
            set => _color = value;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/session/AcceptCreateSessionMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.session;

public class AcceptCreateSessionMessage {

	public static final String TYPE = "AcceptCreateSessionMessage";

	private String sessionID;
	private String color;

	public AcceptCreateSessionMessage() {
	}

	public AcceptCreateSessionMessage(String sessionID,String color) {
		this.sessionID = sessionID;
		this.color = color;
	}

	public String getSessionID() {
		return this.sessionID;
	}

	public void setSessionID(String sessionID) {
		this.sessionID = sessionID;
	}

	public String getColor() {
		return color;
	}

	public void setColor(String color) {
		this.color = color;
	}
}
